openslides/agenda/static/js/agenda/base.js

```diff
@@ -66,6 +75,27 @@ angular.module('OpenSlidesApp.agenda', ['OpenSlidesApp.users'])
                                element.id == self.content_object.id;
                     };
                     return typeof _.findKey(projector.elements, predicate) === 'string';
+                },
+                // project list of speakers
+                projectListOfSpeakers: function() {
+                    return $http.post(
+                        '/rest/core/projector/1/prune_elements/',
+                        [{name: 'agenda/item', id: this.id, list_of_speakers: true}]
```

Comment:
I tested this: This does not work because the list of speakers is projected as agenda/item and needs `name: agenda/item`. I tested e.g.  a motion: this project only the motion but NOT the list of speakers slide of the motion. Therefor you have to send post `{name: "agenda/item", id: 5, list_of_speakers: true}
The current code works good. So why not merge this?